resilience4j-metrics/src/test/java/io/github/resilience4j/metrics/RetryMetricsTest.java

```diff
@@ -58,6 +58,23 @@ public void shouldReturnTotalNumberOfRequestsAs1ForSuccess() {
         assertThat(result).isEqualTo("Success");
     }
 
+    @Test
+    public void shouldReturnTotalNumberOfRequestsAs1ForSuccessVoid() {
+        HelloWorldService helloWorldService = mock(HelloWorldService.class);
+
+        Retry retry = Retry.of("metrics", RetryConfig.<String>custom()
+                .retryOnResult(String::isEmpty)
+                .maxAttempts(5)
+                .build());
+
+
+        retry.executeRunnable(helloWorldService::sayHelloWorld);
+
+        //then(helloWorldService).should().sayHelloWorld();
```

Comment:
why is it commented on, should be?

Reply:
o, yes, pardon, my eyes got used to this line and I forgot to remove it